qtglean/pings.yaml

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# This file defines the pings that are recorded by the Glean SDK.
+# Their code APIs is automatically generated, at build time using,
+# the `glean_parser` PyPI package.
+
+---
+$schema: moz://mozilla.org/schemas/glean/pings/2-0-0
+
+vpnsession:
+  description: |
+    Data for one VPN session, defined as a user turning on and eventually
+    turning off the VPN in the app. This ping will record start/end datestamp,
+    session-based metrics, etc. It will be sent after the user turns off the
+    VPN, and a end datestamp is recorded.
```

Comment:
```suggestion
    VPN.
```

Reply:
The plan is to send it right after the end datestamp is recorded, so this should be accurate.